tokio/Cargo.toml

```diff
@@ -52,14 +52,10 @@ net = [
   "mio/os-ext",
   "mio/net",
   "socket2",
-  "winapi/fileapi",
-  "winapi/handleapi",
-  "winapi/namedpipeapi",
-  "winapi/winbase",
-  "winapi/winnt",
-  "winapi/minwindef",
-  "winapi/accctrl",
-  "winapi/aclapi",
+  "windows-sys/Win32_Foundation",
+  "windows-sys/Win32_Storage_FileSystem",
+  "windows-sys/Win32_System_Pipes",
+  "windows-sys/Win32_System_SystemServices",
```

Comment:
Have you double-checked that this list is not missing anything that is enabled by the Cargo.toml of mio (or another dependency)? We've had breakage before due to mio no longer requiring a feature we needed in Tokio.

Reply:
I checked and found two features that could be an issue - fixed.